src/session.rs

```diff
@@ -458,7 +478,6 @@ impl Session {
         };
         let clock_rate = params.spec().clock_rate;
         let pt = params.pt();
-        let is_repair = pt != header.payload_type;
```

Comment:
Why does this move up?

Reply:
It was part of the revert, it shouldn't have changed